### PR TITLE
feat(ai-training-notebooks): add possibility to attach ssh keys

### PR DESCRIPTION
**Pull request by mael-le-gal**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MLS-2321
| License          | BSD 3-Clause

## Description

On AI Notebooks add the possibility to attach an ssh key when submiting a new notebook

### :flags: Translations

- [x] TRANS-41400

### CODE REVIEW

**Review by JeremyDec — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/notebooks/notebooks.service.js
Comment (mael-le-gal) [reply packages/manager/modules/pci/src/projects/project/notebooks/notebooks.service.js]: fixed

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.controller.js
Comment (mael-le-gal) [reply packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.controller.js]: fixed

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/translations/Messages_fr_FR.json
Comment (mael-le-gal) [reply packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/translations/Messages_fr_FR.json]: fixed

**Comment by JeremyDec** [comment]
Hello @mael-le-gal, your branch needs to be rebase on `develop`. Can you check please?

**Review by JeremyDec — APPROVED** [review]

**Comment by mael-le-gal** [comment]
QA done on this one. Needs approvals before merging.

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.component.js
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.component.js:10]:
```suggestion
```

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.controller.js
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.controller.js:5]: **suggestion:** AFAIK `coreConfig` is not used so it could be removed.

```suggestion
  constructor(NotebookService) {
```

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.controller.js
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.controller.js:9]:
```suggestion
```

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.controller.js
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/notebook-ssh-keys.controller.js:10]:
```suggestion
    // Option indicating if the user wants to add ssh keys
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Review by JeremyDec — DISMISSED** [review]

**Review by antleblanc — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/translations/Messages_fr_FR.json
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/translations/Messages_fr_FR.json:2]:
```suggestion
  "pci_projects_project_training_notebook_add_select_ssh": "Attacher des clés publiques SSH (optionnel)",
```

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/translations/Messages_fr_FR.json
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/translations/Messages_fr_FR.json:3]:
```suggestion
  "pci_projects_project_training_notebook_add_select_ssh_tips_1": "Attacher une clé publique SSH vous permet par la suite d'accéder à distance au notebook via le protocole ssh ou scp",
```

File: packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/translations/Messages_fr_FR.json
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-ssh-keys/translations/Messages_fr_FR.json:8]: replace ssh by `SSH`

File: packages/manager/modules/pci/src/projects/project/notebooks/add/translations/Messages_fr_FR.json
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/notebooks/add/translations/Messages_fr_FR.json:29]: Same as here: https://github.com/ovh/manager/pull/6058/files#r808782968

**Comment by antleblanc** [comment]
Pulling translations.

**Comment by antleblanc** [comment]
Thanks @mohammed-zahaf for all suggestions.
@mael-le-gal Would you take it into in account (on a separate Pull Request)?

**Comment by mohammed-zahaf** [comment]
Ok for me

**Pull request merged**
